app/api/services/users/patch_user.py

```diff
@@ -1,73 +1,10 @@
-from datetime import date, datetime
```

Comment:
git is being unhelpful. I deleted the `user_handler` in favor of `services/users/__init__.py`. This file just contains a lot of the code from the user_handler, so git thinks it was renamed / moved!